test/e2e/pod_create_test.go

```diff
@@ -619,7 +619,10 @@ ENTRYPOINT ["sleep","99999"]
 
 		podCreate = podmanTest.Podman([]string{"pod", "create", "--pid", ns, "--name", podName, "--share", "pid"})
 		podCreate.WaitWithDefaultTimeout()
-		Expect(podCreate).Should(ExitWithError())
+		// FIXME: WEIRD! Two different failure messages:
+		//   container "randomfakeid" not found: no container with name ...
+		//   looking up container to share pid namespace with: no container with name ...
```

Comment:
This was so weird. Race condition maybe?

Reply:
Does anyone know (@cevich maybe) if there's a way to find past Cirrus runs? Debian vs Fedora is one of the first differences I look at when trying to categorize errors, and I remember doing so here, but I'd like to look again.

Reply:
Not really past runs but you can just check the logs from the current run
https://api.cirrus-ci.com/v1/artifact/task/6326849989509120/html/int-podman-fedora-39-root-host-sqlite.log.html
https://api.cirrus-ci.com/v1/artifact/task/5482425059377152/html/int-podman-debian-13-root-host-sqlite.log.html
(just grep for `randomfakeid`)

so yes the diff is between debian/fedora based on selinux condition in the code above

Reply:
Thanks, that's it. What threw me is the f38 failures... because I didn't notice those were *containerized*

Reply:
Way too difficult to conditionalize in tests. I've documented in a comment but left the test as-is.